app/javascript/components/upload/upload-utils.js

```diff
@@ -79,6 +80,14 @@ export function formatFileFromServer(file) {
       raw_counts_associations: []
     }
   }
+  if (file.file_type === 'Gene List' && !file.heatmap_file_info) {
+    // some legacy studies will not have supplemental heatmap file info
+    file.heatmap_file_info = {
+      custom_coloring: false,
+      color_min: -1,
+      color_max: 1
```

Comment:
my understanding is that placeholders != defaults.   Placeholders, if not interacted with, do not actually populate the form.  (e.g. the `Genes e.g. "PTEN NF2"` placeholder in global gene search does not actually search that string if you hit enter).  These are default values, not placeholders, so they should be in black

Reply:
I see your point, though that it's not obvious where we decide to actually populate default values, versus we use empty strings as the default.  Here, I thought since these values directly control the scientific visualization, it would be good to populate them explicitly, so if we ever change the defaults, existing visualizations are not changed

Reply:
Thanks, this clarifies things nicely for me.